tests/src/JIT/Directed/Misc/gettype/gettypetypeofmatrix.csproj

```diff
@@ -22,8 +22,7 @@
     </CodeAnalysisDependentAssemblyPaths>
   </ItemGroup>
   <PropertyGroup>
-    <!-- Set to 'Full' if the Debug? column is marked in the spreadsheet. Leave blank otherwise. -->
-    <DebugType></DebugType>
+    <DebugType>None</DebugType>
```

Comment:
This could be telling us that on Linux we can't write out full PDBs, though it is not clear why we are even trying to write PDBs at all. But even if we are, I would expect a nicer error message than a DLL not found. @vsadov any ideas?

@rartemev Can you look and see what the CSC command line was for those projects with and without this project file edit? It should be in the logs somewhere.